misc/package/Data Files/MWSE/core/mcm/main.lua

```diff
@@ -373,3 +511,8 @@ local function onInitialized()
 	event.trigger("modConfigReady")
 end
 event.register("initialized", onInitialized, { priority = 100 })
+
+event.register('initialized',function (e)
+	timer.start{ callback = loadFavoriteData, duration = 0.5, type = timer.real}
+	loadFavoriteData() -- only need to do it once when the game loads
+end, {priority = -1000000})
```

Comment:
Why is the loading of favorite data done twice? Can it be done in `onInitialized` above?

Reply:
good catch. i think i was doing something weird at the beginning and forgot to change this once things became more stable